[
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     769     gcc: _configtest.c\n770     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     771         3 | #pragma clang diagnostic error \"-Wattributes\"\n     772           |\n     773     _configtest.c: In function '__declspec':\n  >> 774     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "775         5 | int __declspec(thread) foo;\n     776           |                        ^~~\n  >> 777     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     778         9 | {\n     779           | ^\n780     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     781         5 | int __declspec(thread) foo;\n     782           |     ^~~~~~~~~~\n  >> 783     _configtest.c:11: error: expected '{' at end of input\n     784        11 | }\n     785           |",
    "implicit-int]\n     781         5 | int __declspec(thread) foo;\n     782           |     ^~~~~~~~~~\n  >> 783     _configtest.c:11: error: expected '{' at end of input\n     784        11 | }\n     785           |\n786     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     787        11 | }\n     788           |\n     789     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     790         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "784        11 | }\n     785           |\n     786     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     787        11 | }\n     788           |\n789     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     790         3 | #pragma clang diagnostic error \"-Wattributes\"\n     791           |\n     792     _configtest.c: In function '__declspec':\n  >> 793     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "794         5 | int __declspec(thread) foo;\n     795           |                        ^~~\n  >> 796     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     797         9 | {\n     798           | ^\n799     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     800         5 | int __declspec(thread) foo;\n     801           |     ^~~~~~~~~~\n  >> 802     _configtest.c:11: error: expected '{' at end of input\n     803        11 | }\n     804           |",
    "implicit-int]\n     800         5 | int __declspec(thread) foo;\n     801           |     ^~~~~~~~~~\n  >> 802     _configtest.c:11: error: expected '{' at end of input\n     803        11 | }\n     804           |\n805     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     806        11 | }\n     807           |\n     808     failure.\n\n     ...",
    "...\n\n     2874          |                              ^~\n     2875    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2876    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2877     2727 |         if (ret->obmeta.base == -1) {\n     2878          |                              ^~\n     2879    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2899     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2900          |                              ^\n     2901    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2902    numpy/core/src/multiarray/scalartypes.c.src:627:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2903      627 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2904          |     ^~\n\n     ...",
    "...\n\n     3041          |                              ^~\n     3042    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3043    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3044     2727 |         if (ret->obmeta.base == -1) {\n     3045          |                              ^~\n     3046    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     95      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 96      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 97      collect2: error: ld returned 1 exit status\n     98      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 99      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 100     collect2: error: ld returned 1 exit status",
    ">> 97      collect2: error: ld returned 1 exit status\n     98      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 99      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 100     collect2: error: ld returned 1 exit status\n     101     failure.\n     102     removing: _configtest.c _configtest.o\n     103     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     104",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     253     gcc: _configtest.c\n     254     _configtest.c: In function 'main':\n>> 255     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     256         7 |     (void) SIZEOF_LONGDOUBLE;\n     257           |            ^~~~~~~~~~~~~~~~~\n     258           |            SIZEOF_LONG_DOUBLE\n     259     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "256         7 |     (void) SIZEOF_LONGDOUBLE;\n     257           |            ^~~~~~~~~~~~~~~~~\n     258           |            SIZEOF_LONG_DOUBLE\n     259     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     260     _configtest.c: In function 'main':\n>> 261     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     262         7 |     (void) SIZEOF_LONGDOUBLE;\n     263           |            ^~~~~~~~~~~~~~~~~\n     264           |            SIZEOF_LONG_DOUBLE\n     265     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     399     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 400     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 401     collect2: error: ld returned 1 exit status\n     402     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 403     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 404     collect2: error: ld returned 1 exit status",
    ">> 401     collect2: error: ld returned 1 exit status\n     402     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 403     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 404     collect2: error: ld returned 1 exit status\n     405     failure.\n     406     removing: _configtest.c _configtest.o\n     407     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     408",
    "769     gcc: _configtest.c\n     770     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     771         3 | #pragma clang diagnostic error \"-Wattributes\"\n     772           |\n     773     _configtest.c: In function '__declspec':\n>> 774     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     775         5 | int __declspec(thread) foo;\n     776           |                        ^~~\n  >> 777     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     778         9 | {",
    "772           |\n     773     _configtest.c: In function '__declspec':\n  >> 774     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     775         5 | int __declspec(thread) foo;\n     776           |                        ^~~\n>> 777     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     778         9 | {\n     779           | ^\n     780     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     781         5 | int __declspec(thread) foo;",
    "778         9 | {\n     779           | ^\n     780     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     781         5 | int __declspec(thread) foo;\n     782           |     ^~~~~~~~~~\n>> 783     _configtest.c:11: error: expected '{' at end of input\n     784        11 | }\n     785           |\n     786     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     787        11 | }\n     788           |",
    "788           |\n     789     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     790         3 | #pragma clang diagnostic error \"-Wattributes\"\n     791           |\n     792     _configtest.c: In function '__declspec':\n>> 793     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     794         5 | int __declspec(thread) foo;\n     795           |                        ^~~\n  >> 796     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     797         9 | {",
    "791           |\n     792     _configtest.c: In function '__declspec':\n  >> 793     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     794         5 | int __declspec(thread) foo;\n     795           |                        ^~~\n>> 796     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     797         9 | {\n     798           | ^\n     799     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     800         5 | int __declspec(thread) foo;",
    "797         9 | {\n     798           | ^\n     799     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     800         5 | int __declspec(thread) foo;\n     801           |     ^~~~~~~~~~\n>> 802     _configtest.c:11: error: expected '{' at end of input\n     803        11 | }\n     804           |\n     805     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     806        11 | }\n     807           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     1135    gcc: _configtest.c\n     1136    _configtest.c: In function 'main':\n>> 1137    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1138        8 |     (void) HAVE_DECL_SIGNBIT;\n     1139          |            ^~~~~~~~~~~~~~~~~\n     1140          |            HAVE_DECL_ISNAN\n     1141    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1138        8 |     (void) HAVE_DECL_SIGNBIT;\n     1139          |            ^~~~~~~~~~~~~~~~~\n     1140          |            HAVE_DECL_ISNAN\n     1141    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1142    _configtest.c: In function 'main':\n>> 1143    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1144        8 |     (void) HAVE_DECL_SIGNBIT;\n     1145          |            ^~~~~~~~~~~~~~~~~\n     1146          |            HAVE_DECL_ISNAN\n     1147    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1725    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1726    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1727    collect2: error: ld returned 1 exit status\n     1728    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1729    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1730    collect2: error: ld returned 1 exit status",
    ">> 1727    collect2: error: ld returned 1 exit status\n     1728    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1729    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1730    collect2: error: ld returned 1 exit status\n     1731    failure.\n     1732    removing: _configtest.c _configtest.o\n     1733    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1734",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2877     2727 |         if (ret->obmeta.base == -1) {\n     2878          |                              ^~\n     2879    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2880    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2881     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2882          |                       ^\n  >> 2883    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2879    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2880    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2881     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2882          |                       ^\n>> 2883    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2885          |                        ^\n  >> 2886    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2887     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2881     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2882          |                       ^\n  >> 2883    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2885          |                        ^\n>> 2886    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2887     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2888          |                         ^\n  >> 2889    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2890     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2885          |                        ^\n  >> 2886    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2887     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2888          |                         ^\n>> 2889    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2890     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2891          |                             ^\n  >> 2892    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2888          |                         ^\n  >> 2889    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2890     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2891          |                             ^\n>> 2892    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2893     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2894          |                         ^\n  >> 2895    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2896     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2891          |                             ^\n  >> 2892    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2893     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2894          |                         ^\n>> 2895    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2896     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2897          |                          ^\n  >> 2898    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2899     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2893     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2894          |                         ^\n  >> 2895    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2896     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2897          |                          ^\n>> 2898    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2899     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2900          |                              ^\n     2901    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3044     2727 |         if (ret->obmeta.base == -1) {\n     3045          |                              ^~\n     3046    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3047    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3048     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3049          |                       ^\n  >> 3050    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3046    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3047    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3048     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3049          |                       ^\n>> 3050    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3052          |                        ^\n  >> 3053    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3054     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3048     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3049          |                       ^\n  >> 3050    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3052          |                        ^\n>> 3053    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3054     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3055          |                         ^\n  >> 3056    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3057     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3052          |                        ^\n  >> 3053    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3054     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3055          |                         ^\n>> 3056    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3057     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3058          |                             ^\n  >> 3059    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3055          |                         ^\n  >> 3056    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3057     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3058          |                             ^\n>> 3059    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3060     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3061          |                         ^\n  >> 3062    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3063     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3058          |                             ^\n  >> 3059    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3060     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3061          |                         ^\n>> 3062    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3063     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3064          |                          ^\n  >> 3065    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3066     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3060     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3061          |                         ^\n  >> 3062    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3063     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3064          |                          ^\n>> 3065    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3066     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3067          |                              ^\n  >> 3068    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     769     gcc: _configtest.c\n770     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     771         3 | #pragma clang diagnostic error \"-Wattributes\"\n     772           |\n     773     _configtest.c: In function '__declspec':\n  >> 774     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "775         5 | int __declspec(thread) foo;\n     776           |                        ^~~\n  >> 777     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     778         9 | {\n     779           | ^\n780     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     781         5 | int __declspec(thread) foo;\n     782           |     ^~~~~~~~~~\n  >> 783     _configtest.c:11: error: expected '{' at end of input\n     784        11 | }\n     785           |",
    "implicit-int]\n     781         5 | int __declspec(thread) foo;\n     782           |     ^~~~~~~~~~\n  >> 783     _configtest.c:11: error: expected '{' at end of input\n     784        11 | }\n     785           |\n786     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     787        11 | }\n     788           |\n     789     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     790         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "784        11 | }\n     785           |\n     786     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     787        11 | }\n     788           |\n789     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     790         3 | #pragma clang diagnostic error \"-Wattributes\"\n     791           |\n     792     _configtest.c: In function '__declspec':\n  >> 793     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "794         5 | int __declspec(thread) foo;\n     795           |                        ^~~\n  >> 796     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     797         9 | {\n     798           | ^\n799     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     800         5 | int __declspec(thread) foo;\n     801           |     ^~~~~~~~~~\n  >> 802     _configtest.c:11: error: expected '{' at end of input\n     803        11 | }\n     804           |",
    "implicit-int]\n     800         5 | int __declspec(thread) foo;\n     801           |     ^~~~~~~~~~\n  >> 802     _configtest.c:11: error: expected '{' at end of input\n     803        11 | }\n     804           |\n805     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     806        11 | }\n     807           |\n     808     failure.\n\n     ...",
    "...\n\n     2874          |                              ^~\n     2875    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2876    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2877     2727 |         if (ret->obmeta.base == -1) {\n     2878          |                              ^~\n     2879    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2899     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2900          |                              ^\n     2901    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2902    numpy/core/src/multiarray/scalartypes.c.src:627:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2903      627 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2904          |     ^~\n\n     ...",
    "...\n\n     3041          |                              ^~\n     3042    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3043    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3044     2727 |         if (ret->obmeta.base == -1) {\n     3045          |                              ^~\n     3046    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     95      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 96      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 97      collect2: error: ld returned 1 exit status\n     98      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 99      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 100     collect2: error: ld returned 1 exit status",
    ">> 97      collect2: error: ld returned 1 exit status\n     98      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 99      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 100     collect2: error: ld returned 1 exit status\n     101     failure.\n     102     removing: _configtest.c _configtest.o\n     103     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     104",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     253     gcc: _configtest.c\n     254     _configtest.c: In function 'main':\n>> 255     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     256         7 |     (void) SIZEOF_LONGDOUBLE;\n     257           |            ^~~~~~~~~~~~~~~~~\n     258           |            SIZEOF_LONG_DOUBLE\n     259     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "256         7 |     (void) SIZEOF_LONGDOUBLE;\n     257           |            ^~~~~~~~~~~~~~~~~\n     258           |            SIZEOF_LONG_DOUBLE\n     259     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     260     _configtest.c: In function 'main':\n>> 261     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     262         7 |     (void) SIZEOF_LONGDOUBLE;\n     263           |            ^~~~~~~~~~~~~~~~~\n     264           |            SIZEOF_LONG_DOUBLE\n     265     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     399     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 400     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 401     collect2: error: ld returned 1 exit status\n     402     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 403     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 404     collect2: error: ld returned 1 exit status",
    ">> 401     collect2: error: ld returned 1 exit status\n     402     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 403     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 404     collect2: error: ld returned 1 exit status\n     405     failure.\n     406     removing: _configtest.c _configtest.o\n     407     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     408",
    "769     gcc: _configtest.c\n     770     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     771         3 | #pragma clang diagnostic error \"-Wattributes\"\n     772           |\n     773     _configtest.c: In function '__declspec':\n>> 774     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     775         5 | int __declspec(thread) foo;\n     776           |                        ^~~\n  >> 777     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     778         9 | {",
    "772           |\n     773     _configtest.c: In function '__declspec':\n  >> 774     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     775         5 | int __declspec(thread) foo;\n     776           |                        ^~~\n>> 777     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     778         9 | {\n     779           | ^\n     780     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     781         5 | int __declspec(thread) foo;",
    "778         9 | {\n     779           | ^\n     780     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     781         5 | int __declspec(thread) foo;\n     782           |     ^~~~~~~~~~\n>> 783     _configtest.c:11: error: expected '{' at end of input\n     784        11 | }\n     785           |\n     786     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     787        11 | }\n     788           |",
    "788           |\n     789     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     790         3 | #pragma clang diagnostic error \"-Wattributes\"\n     791           |\n     792     _configtest.c: In function '__declspec':\n>> 793     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     794         5 | int __declspec(thread) foo;\n     795           |                        ^~~\n  >> 796     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     797         9 | {",
    "791           |\n     792     _configtest.c: In function '__declspec':\n  >> 793     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     794         5 | int __declspec(thread) foo;\n     795           |                        ^~~\n>> 796     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     797         9 | {\n     798           | ^\n     799     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     800         5 | int __declspec(thread) foo;",
    "797         9 | {\n     798           | ^\n     799     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     800         5 | int __declspec(thread) foo;\n     801           |     ^~~~~~~~~~\n>> 802     _configtest.c:11: error: expected '{' at end of input\n     803        11 | }\n     804           |\n     805     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     806        11 | }\n     807           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     1135    gcc: _configtest.c\n     1136    _configtest.c: In function 'main':\n>> 1137    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1138        8 |     (void) HAVE_DECL_SIGNBIT;\n     1139          |            ^~~~~~~~~~~~~~~~~\n     1140          |            HAVE_DECL_ISNAN\n     1141    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1138        8 |     (void) HAVE_DECL_SIGNBIT;\n     1139          |            ^~~~~~~~~~~~~~~~~\n     1140          |            HAVE_DECL_ISNAN\n     1141    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1142    _configtest.c: In function 'main':\n>> 1143    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1144        8 |     (void) HAVE_DECL_SIGNBIT;\n     1145          |            ^~~~~~~~~~~~~~~~~\n     1146          |            HAVE_DECL_ISNAN\n     1147    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1725    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1726    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1727    collect2: error: ld returned 1 exit status\n     1728    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1729    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1730    collect2: error: ld returned 1 exit status",
    ">> 1727    collect2: error: ld returned 1 exit status\n     1728    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1729    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.3-v2zf6rxe3ynl\n             lvl375ud3decfpum3b7o/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1730    collect2: error: ld returned 1 exit status\n     1731    failure.\n     1732    removing: _configtest.c _configtest.o\n     1733    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1734",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2877     2727 |         if (ret->obmeta.base == -1) {\n     2878          |                              ^~\n     2879    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2880    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2881     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2882          |                       ^\n  >> 2883    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2879    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2880    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2881     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2882          |                       ^\n>> 2883    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2885          |                        ^\n  >> 2886    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2887     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2881     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2882          |                       ^\n  >> 2883    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2885          |                        ^\n>> 2886    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2887     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2888          |                         ^\n  >> 2889    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2890     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2884     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2885          |                        ^\n  >> 2886    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2887     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2888          |                         ^\n>> 2889    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2890     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2891          |                             ^\n  >> 2892    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2888          |                         ^\n  >> 2889    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2890     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2891          |                             ^\n>> 2892    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2893     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2894          |                         ^\n  >> 2895    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2896     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2891          |                             ^\n  >> 2892    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2893     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2894          |                         ^\n>> 2895    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2896     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2897          |                          ^\n  >> 2898    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2899     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2893     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2894          |                         ^\n  >> 2895    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2896     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2897          |                          ^\n>> 2898    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2899     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2900          |                              ^\n     2901    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3044     2727 |         if (ret->obmeta.base == -1) {\n     3045          |                              ^~\n     3046    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3047    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3048     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3049          |                       ^\n  >> 3050    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3046    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3047    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3048     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3049          |                       ^\n>> 3050    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3052          |                        ^\n  >> 3053    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3054     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3048     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3049          |                       ^\n  >> 3050    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3052          |                        ^\n>> 3053    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3054     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3055          |                         ^\n  >> 3056    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3057     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3051     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3052          |                        ^\n  >> 3053    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3054     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3055          |                         ^\n>> 3056    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3057     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3058          |                             ^\n  >> 3059    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3055          |                         ^\n  >> 3056    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3057     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3058          |                             ^\n>> 3059    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3060     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3061          |                         ^\n  >> 3062    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3063     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3058          |                             ^\n  >> 3059    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3060     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3061          |                         ^\n>> 3062    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3063     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3064          |                          ^\n  >> 3065    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3066     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3060     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3061          |                         ^\n  >> 3062    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3063     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3064          |                          ^\n>> 3065    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3066     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3067          |                              ^\n  >> 3068    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -"
]